extension/chrome/settings/inbox/inbox.ts

```diff
@@ -132,6 +154,29 @@ export class InboxView extends View {
     Xss.sanitizeRender('h1', title);
   };
 
+  private preRenderSecureComposeInFullScreen = () => {
+    if (this.useFullScreenSecureCompose && this.thunderbirdMsgId) {
+      this.injector.openComposeWin(undefined, true, this.thunderbirdMsgId, this.composeMethod);
+    }
+  };
+
+  private parseThreadIdFromHeaderMessageId = async () => {
+    let threadId;
+    const tbMsgId = Number(this.thunderbirdMsgId);
+    if (Catch.isThunderbirdMail() && tbMsgId && !this.useFullScreenSecureCompose) {
+      const thunderbirdFullMsg = await browser.messages.getFull(tbMsgId);
+      if (thunderbirdFullMsg.headers) {
+        const messageId = thunderbirdFullMsg.headers['message-id'][0];
+        if (messageId) {
+          const gmail = new Gmail(this.acctEmail);
+          const gmailRes = await gmail.threadIdGet(messageId);
+          threadId = gmailRes.messages[0].threadId;
```

Comment:
we already have `new Gmail` initialized, can just use it:
```ts
const gmailRes = await this.gmail.threadIdGet(messageId);
```

Reply:
Ah yes, will do it as as follows.